### PR TITLE
GC unused code.

### PR DESCRIPTION
**Pull request by baldersheim**
@havardpe PR
@vekterli FYI

### CODE REVIEW

File: fastos/src/vespa/fastos/app.cpp [hunk 1]
```diff
@@ -55,34 +54,18 @@ bool FastOS_ApplicationInterface::Init ()
...
-                }
-                else
-                {
-                    fprintf(stderr,
-                            "FastOS_Socket::InitializeServices() returned:\n[%s]\n",
-                            errorMsg);
-                }
-            }
-            else
+            } else
```
Comment (havardpe) [inline fastos/src/vespa/fastos/app.cpp]: prefer braces also for single-expression blocks

File: fastos/src/vespa/fastos/app.cpp [hunk 2]
```diff
@@ -55,34 +54,18 @@ bool FastOS_ApplicationInterface::Init ()
...
                 fprintf(stderr, "FastOS_File class initialization failed.\n");
-        }
-        else
+        } else
```
Comment (havardpe) [inline fastos/src/vespa/fastos/app.cpp]: prefer braces also for single-expression blocks

File: fastos/src/vespa/fastos/app.cpp [hunk 3]
```diff
@@ -55,34 +54,18 @@ bool FastOS_ApplicationInterface::Init ()
...
             fprintf(stderr, "FastOS_Thread class initialization failed.\n");
-    }
-    else
+    } else
```
Comment (havardpe) [inline fastos/src/vespa/fastos/app.cpp]: prefer braces also for single-expression blocks

**Review by havardpe — APPROVED** [review]

File: fastos/src/vespa/fastos/app.cpp (hunk 1 above)
Comment (baldersheim) [reply fastos/src/vespa/fastos/app.cpp]: Next time, if not remove gets there first.....

File: fastos/src/vespa/fastos/app.cpp (hunk 3 above)
Comment (baldersheim) [reply fastos/src/vespa/fastos/app.cpp]: Next time, if not remove gets there first.....

File: fastos/src/vespa/fastos/app.cpp (hunk 2 above)
Comment (baldersheim) [reply fastos/src/vespa/fastos/app.cpp]: Next time, if not remove gets there first.....

**Pull request merged**